include/circt/Dialect/Calyx/CalyxPrimitives.td

```diff
@@ -335,7 +335,7 @@ def XorLibOp  : CombinationalArithBinaryLibraryOp<"xor"> {}
 class ArithBinaryFloatingPointLibraryOp<string mnemonic> : ArithBinaryLibraryOp<mnemonic, [
   SameTypeConstraint<"left", "out">]> {}
 
-def AddFNOp : ArithBinaryFloatingPointLibraryOp<"addFN"> {
+def AddFNOp : ArithBinaryFloatingPointLibraryOp<"ieee754.add"> {
```

Comment:
Why does this get printed as `calyx.std_ieee754.add` instead of `calyx.ieee754.add`? Can we make it be the latter please?

Reply:
Sure, no problem
Because `ArithBinaryFloatingPointLibraryOp` inherits `CalyxPrimitive`, which automatically prints op names as `"std_"+`, [here](https://github.com/llvm/circt/blob/8a270d72f2e70c1db80db8184ad2ba3a89403d38/include/circt/Dialect/Calyx/CalyxPrimitives.td#L261)